src/components/CreateCoursePage/index.jsx

```diff
@@ -164,6 +171,14 @@ class CreateCoursePage extends React.Component {
           { showForm
           && (
             <div>
+              {errorArray.length > 1 && (
```

Comment:
What would `errorArray` contain if length is 1?

Reply:
don't know...essentially this seems to be some sort of condition for displaying an error message....i have only changed the position of where the error message is being displayed....

if this ^ doesn't make sense, just look at the complete file and hopefully it will be clear